examples/roads2png.rs

```diff
@@ -73,66 +39,57 @@ impl Image {
     }
 }
 
-#[derive(Debug, Clone)]
-struct MapTransform {
-    min_x: f64,
-    max_y: f64,
-    scale_x: f64,
-    scale_y: f64,
+fn compute_bounds<'a>(mut iter: impl Iterator<Item = RefNode<'a>>) -> ((i64, i64), (i64, i64)) {
```

Comment:
I have to admin I prefer to have a type for `GeoCoord` instead of pairs.

Reply:
Makes sense, I can restore this